src/math/crypto/curve/ec_point.zig

```diff
@@ -152,6 +163,18 @@ pub const AffinePoint = struct {
     y: Felt252,
     infinity: bool,
 
+    /// Initializes an `AffinePoint` with the specified x and y coordinates.
+    ///
+    /// # Parameters
+    /// - `x`: The x-coordinate of the point.
+    /// - `y`: The y-coordinate of the point.
+    ///
+    /// # Returns
+    /// An initialized `ECPoint` with the provided coordinates.
+    pub fn init(x: Felt252, y: Felt252, infinity: bool) Self {
```

Comment:
Here better to call it `initUnchecked` as long as we don't check that the provided coordinates give a point that is on the curve.